js/exporter/jspdf/v3/pdf_utils_v3.js

```diff
@@ -61,19 +63,27 @@ function drawRect(doc, x, y, width, height, style) {
     }
 }
 
-function drawTextInRect(doc, text, rect, wordWrapEnabled, jsPdfTextOptions) {
-    const textArray = getTextLines(doc, text, doc.getFont(), { wordWrapEnabled, columnWidth: rect.w });
+function drawTextInRect(doc, text, padding, rect, wordWrapEnabled, jsPdfTextOptions) {
+    const textArray = getTextLines(doc, text, padding, doc.getFont(), { wordWrapEnabled, columnWidth: rect.w });
     const linesCount = textArray.length;
 
-    const heightOfOneLine = calculateTextHeight(doc, textArray[0], doc.getFont(), { wordWrapEnabled: false });
+    const heightOfOneLine = calculateTextHeight(doc, textArray[0], 0, doc.getFont(), { wordWrapEnabled: false });
+    const contentRect = isDefined(padding)
```

Comment:
I think it's better to introduce cell padding variable as in this comment https://github.com/DevExpress/DevExtreme/pull/19235/files#r709812183

Something like this:

```js
const cellPadding = padding ?? 0;
const textRect = {
            x: rect.x + cellPadding ,
            y: rect.y + cellPadding ,
            w: rect.w - cellPadding  * 2,
            h: rect.h - cellPadding  * 2
        }
```

Ant then use textRect variable